nestkernel/connection_manager.cpp

```diff
@@ -1288,33 +1315,35 @@ nest::ConnectionManager::get_sources( std::vector< index > targets,
 void
 nest::ConnectionManager::get_targets( std::vector< index > sources,
```

Comment:
Since `sources` is not modified inside the method, this should be `const std::vector< index>& sources` to avoid unnecessary copying.